src/main/java/uk/gov/digital/ho/hocs/queue/migration/MigrationFailureService.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.digital.ho.hocs.queue.migration;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class MigrationFailureService {
```

Comment:
I'm not sure this is needed as this data could be stored in the message_log table?

Reply:
For me the status in the message_log table should say failed, we should log out appropriately the error and then we can cross reference to remediate